### PR TITLE
update information for insyde

### PR DESCRIPTION
**Pull request by eebssk1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,22 @@ Locate `Demo_SPC_Code_Signing.cer` and `Demo_CA_Root.cer` on your `USB`, pressin
...
+
+Scroll down and choose button includes "DB" (not DBX!!)
+
+Click "Enroll"
+
+Choose the cer file using the file browser(you can select only one, so you need to do it again for the other cer files)
+
+When it asks about Owner GUID, you can use anyone you like, just keep sure it's same across your cer files.
+
+Click okay and then "Exit saving changes" in the bottom.
```
Comment (HypsyNZ) [inline README.md:65]: > then "Exit saving changes" 

I'm going to need to see proof that this doesn't brick your motherboard (video)

File: README.md [hunk 2]
```diff
@@ -48,6 +48,22 @@ Locate `Demo_SPC_Code_Signing.cer` and `Demo_CA_Root.cer` on your `USB`, pressin
 
 The second time you press enter you'll be prompted to confirm you want to update the Certificate Store, Select `Yes`.
 
+------
+
+For some (subsets of) Insyde based UEFI, check the following.
+
+Click "Administrate Secure Boot" in the main menu.
```
Comment (HypsyNZ) [inline README.md:55]: Can you provide a screenshot of all the menus you mention

File: README.md [hunk 3]
```diff
@@ -48,6 +48,22 @@ Locate `Demo_SPC_Code_Signing.cer` and `Demo_CA_Root.cer` on your `USB`, pressin
...
+
+Scroll down and choose button includes "DB" (not DBX!!)
+
+Click "Enroll"
+
+Choose the cer file using the file browser(you can select only one, so you need to do it again for the other cer files)
+
+When it asks about Owner GUID, you can use anyone you like, just keep sure it's same across your cer files.
```
Comment (HypsyNZ) [inline README.md:63]: Please elaborate on this

**Review by HypsyNZ — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (eebssk1) [reply README.md:55]: hold on

File: README.md (hunk 3 above)
Comment (eebssk1) [reply README.md:63]: it's actually showed in your step 5 screenshot. But insyde is too lazy to generate for you and some oem preinstalled their cert may have different owner GUID.
https://techlibrary.hpe.com/docs/iss/proliant-gen10-uefi/GUID-E4427875-D123-4BBF-9056-342168478A02.html

**Comment by HypsyNZ** [comment]
Please add more detailed instructions or explain how this pull request adds new information, please don't send links.

**Comment by eebssk1** [comment]
Tested. Unfortunately, this does not work on my device. >_<

**Pull request closed without merging**